test/test_inputs.py

```diff
@@ -315,16 +311,6 @@ def test_in_interface(self):
         self.assertEqual(iface.process([["a", "c"]])[0], ["a|c"])
         self.assertEqual(iface.process([[]])[0], [""])
         checkboxes_input = gr.inputs.CheckboxGroup(["a", "b", "c"], type="index")
-        iface = gr.Interface(
```

Comment:
Hmm, why did we remove these checks?

Reply:
Because we no longer have interpretation for textbox output, as per this comment: https://github.com/gradio-app/gradio/pull/765#discussion_r830451228